src/views/index.html

```diff
@@ -77,6 +77,7 @@
     <nav id="main-menu" class="hidden">
       <ul>
         <li><a id="things-menu-item" href="/things" class="selected">Things</a></li>
+        <li><a id="rules-engine-menu-item" href="/rules-engine">Rules Engine</a></li>
```

Comment:
FWIW I've started on an Experiments section for Settings in https://github.com/mozilla-iot/gateway/pull/254 - unless you find a way to automatically detect rules engine for now.